streams.json

```diff
@@ -6758,141 +8900,189 @@
-          "gav": "org.picketbox",
-          "comment": ""
+          "gav": "picketbox-commons:org.picketbox:1.0.0.final-redhat-5",
+          "comment": "TAG suspicious. TAG in pom: jboss-parent-16, TAG in mapping tool: 1.0.0.final. "
         },
         {
           "component_name": "picketlink-bindings-25",
           "contacts": [
               "pskopek@redhat.com"
           ],
-          "repository_type": "GIT",
-          "repository_url": "https://github.com/jbossas/redhat-picketlink-bindings",
+          "repository_type": "",
+          "repository_url": "",
           "codebase": "eap-7.x",
-          "tag": "v2.5.5.SP8",
-          "version": "2.5.5.SP8-redhat-1",
-          "gav": "org.picketlink.distribution",
+          "tag": "",
+          "group_id": "",
+          "artifact_id": "",
+          "version": "",
+          "gav": "",
+          "comment": "TAG not found. "
+        },
+        {
+          "component_name": "projectodd-vdx",
+          "contacts": [],
+          "repository_type": "GIT",
+          "repository_url": "https://github.com/projectodd/vdx.git",
+          "codebase": "",
+          "tag": "1.1.6",
+          "group_id": "org.projectodd.vdx",
+          "artifact_id": "vdx-parent",
+          "version": "1.1.6.redhat-1",
+          "gav": "vdx-parent:org.projectodd.vdx:1.1.6.redhat-1",
           "comment": ""
         },
         {
           "component_name": "relaxng-datatype-java",
           "contacts": [],
           "repository_type": "GIT",
-          "repository_url": "git://git.app.eng.bos.redhat.com/relaxng-datatype-java",
+          "repository_url": "git://git.app.eng.bos.redhat.com/relaxng-datatype-java.git",
           "codebase": "",
-          "tag": "2011.1.0.redhat-10",
+          "tag": "relaxngDatatype-2011.1",
+          "group_id": "com.github.relaxng",
+          "artifact_id": "relaxngDatatype",
           "version": "2011.1.0.redhat-10",
-          "gav": "com.github.relaxng",
+          "gav": "relaxngDatatype:com.github.relaxng:2011.1.0.redhat-10",
           "comment": " No upstream, only repo is git.app ?"
         },
         {
           "component_name": "resteasy",
           "contacts": [
               "alessio.soldano@jboss.com",
-              "bburke@redhat.com"
+            "bburke@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/resteasy/resteasy",
+          "repository_url": "https://github.com/resteasy/Resteasy.git",
           "codebase": "3.0",
           "tag": "3.0.24.Final",
+          "group_id": "org.jboss.resteasy",
+          "artifact_id": "resteasy-jaxrs-all",
           "version": "3.0.24.Final-redhat-1",
-          "gav": "org.jboss.resteasy",
+          "gav": "resteasy-jaxrs-all:org.jboss.resteasy:3.0.24.Final-redhat-1",
           "comment": " There is https://github.com/jbossas/redhat-resteasy but it is dead?"
         },
         {
           "component_name": "rhino-js",
```

Comment:
It looks like we are not using this dependency, I would suggest leaving it out of the final list.

Reply:
+1 seems unused.